dataminer/Functions/Video_Thumbnails/Configuring_Video_Thumbnails/Allowed_paths_connection_DataMiner_proxy.md

```diff
@@ -0,0 +1,35 @@
+---
+uid: Allowed_paths_connection_DataMiner_proxy
+---
+
+# Allowed paths in case of connection via DataMiner proxy
+
+If a client does not have direct access to the video server, it can request a video feed via the DataMiner Agent acting as a proxy server.
+
+When a video feed is requested via proxy, the list of allowed URLs will be restricted for security reasons.
+
+By default, the following URLs are allowed:
+
+```txt
+http://ipaddress/axis-cgi/...
+http://ipaddress/video_thumbnails/...
+http://ipaddress/tile/...
+http://ipaddress/images/getthumbnail/...
```

Comment:
not entirely correct, this is the correct list:

* http://ipaddress/axis-cgi/...
* http://ipaddress/video_thumbnails/...
* http://ipaddress/tile...
* http://ipaddress/images/thumbs/...
* http://ipaddress/cgi-bin/getthumbnail...
* http://ipaddress/tmp/JPEG_Thumbnail/...
* http://ipaddress/control/frame/slot/getThumbImage...